src/components/asset-list/RecyclerAssetList2/core/RowRenderer.tsx

```diff
@@ -51,6 +51,7 @@ function rowRenderer(type: CellType, { uid }: { uid: string }) {
             );
           case CellType.ASSETS_HEADER:
             return (
+              // @ts-expect-error JavaScript component
```

Comment:
This was needed because `magicMemo` was converted to TypeScript, so it's now properly forwarding through the types from `AssetListHeader` but the types aren't able to be inferred correctly because it's a JavaScript component.